django_refugeerights/metricsdownloader/tasks.py

```diff
@@ -0,0 +1,56 @@
+# metric = "stores.refugeerights_qa.total.ussd.sessions.transient.sum"
+from datetime import datetime
+from celery import task
+from django.conf import settings
+from go_http.metrics import MetricsApiClient
+from .models import Metric, MetricRecord
+
+
+def get_date():
+    today = datetime.today()
+    # use the same time for all metrics
+    date = datetime(today.year, today.month, today.day)
+    return date
+
+
+@task()
+def record_all_metrics():
+    """
+    Task to record all metricstore values at a point in time
+    """
+    metrics = Metric.objects.all()
+    num_metrics = 0
+
+    for metric in metrics:
+        num_metrics += 1
+        fetch_metric.delay(metric)
+    return "Recorded values for %d metrics" % num_metrics
+
+
+@task()
+def fetch_metric(metric):
+    """
+    Task to look up a metric value from the metricstore
+    """
+    metric_client = MetricsApiClient(settings.METRIC_API_KEY)
+    response = metric_client.get_metric(
+        metric.lookup_name,
+        "-" + settings.METRIC_RECORDING_INTERVAL,
+        settings.METRIC_RECORDING_INTERVAL,
+        "")
+    metric_value = float(response.values()[0][-1]['y'])
```

Comment:
Hmm, do you think we should record 0.0 or rather not record anything?  I'm leaning towards not recording anything.